src/main/java/org/aksw/gerbil/dataset/impl/derczysnki/DerczynskiDataset.java

```diff
@@ -82,6 +82,11 @@ public void init() throws GerbilException {
 				tweet.append(line + "\n");
 				line = reader.readLine();
 			}
+			// Get Markings
```

Comment:
Shouldn't there be a check whether there is a tweet to be added? Otherwise, a file that ends with an empty line would lead to the situation that a tweet is added two times.

Reply:
Yes, I added a check for that, thanks. It should even cause an error because the StringBuilder would be empty in that case.